conventions/Documentation.md

```diff
@@ -34,34 +34,67 @@ You can provide code snippets demonstrating a specific behavior or pattern for a
 
 ## Storybook
 
-Calcite Components uses [Storybook](https://storybook.js.org/) to provide an interactive showcase of components with accompanying documentation.
-
-For each main component (i.e. one that can be used by itself), there should be a `<component-name>.stories.ts` file in its component folder.
-
-Each story should provide access to relevant [knobs](https://github.com/storybookjs/storybook/tree/next/addons/knobs) so users can test out different properties.
+Calcite Components uses [Storybook](https://storybook.js.org/) to provide an interactive showcase of components with accompanying documentation and also to provide previews for screenshot testing.
 
 ### Writing stories
 
+For each main component (i.e. one that can be used by itself), there should be a `<component-name>.stories.ts` file in its component folder.
+
 Each component should use the [Component Story Format (CSF)](https://storybook.js.org/docs/html/api/csf) with at least one story. If your component has properties that effect visual styles, you can use the [storybook knobs addon](https://www.npmjs.com/package/@storybook/addon-knobs) to allow people to manipulate properties and see live updates in the documentation. A minimal stories file might look something like this:
 
-```
+```ts
 import { storiesOf } from '@storybook/html';
 import { boolean } from '@storybook/addon-knobs'
 import notes from './readme.md';
 
 export default {
-  title: "Components/X",
+  title: "Components/My component",
   parameters: { notes }
 };
 
-storiesOf('My component', module)
-  .add('Simple', () => `
-    <my-component demo-prop="${boolean("demo-prop", true)}"></my-component>
-  , { notes })`
+export function simple = (): string => html`
+  <my-component demo-prop="${boolean("demo-prop", true)}"></my-component>
+`;
 ```
 
 Make sure to import the README generated by Stencil so that it can be viewed in Storybook. Storybook is deployed during every `next` version, whereas as the SDK site is only updated after `beta` releases. Storybook is a great resource for testing the newest changes and viewing the newest API reference.
 
+The following are guidelines for adding/updating stories:
+
+#### General
+
+- Story names should be camelCased
+- All stories need to import the `storyFilters` util and include it in the default export so stories can be properly filtered
+  ```ts
+  export default {
+    title: "Components/My Component",
+    parameters: {
+      notes: readme
+    },
+    ...storyFilters()
+  };
+  ```
+- Update the [main custom theme](https://github.com/Esri/calcite-components/blob/master/src/05-custom-theme.stories.mdx) story instead of adding a specific story showing how to use a custom CSS prop
+- Should only have HTML for the component or use case itself (e.g., no need to wrap in calcite-label)
```

Comment:
☝️ I'm wondering if this matters for the Storybook a11y tab to function properly or not?  Maybe that's not a concern on this storybook anymore?

Reply:
I don't think we really use or used the a11y tab so I'm not against removing it if we want to.

Reply:
May be worth mentioning this somewhere in the a11y conventions doc so we can point to it if people say "Storybook says you're components aren't accessible" cc @geospatialem

Reply:
I meant we should mention that the stories may not be accessible since we aren't adding labels. So I think this isn't true anymore:

> As new components and enhancements are added, ensure stories are updated to test accessibility. This includes as properties are added to components, to ensure we're upholding high standards to fit our audience's needs.

But correct me if I'm wrong @driskull @jcfranco